src/Utility/DataExporter.h

```diff
@@ -192,6 +192,12 @@ class DataWriter
   //! \param[in] level Level we just wrote to the file
   virtual void closeFile(int level) = 0;
 
+  //! \brief Prepare writer for writing a time level.
+  //! \details Used by SVV where we have to preallocate datasets
+  virtual bool prepare(int level, const DataEntry& entry,
+                       bool geometryUpdated, const std::string& prefix)
+  { return true; }
```

Comment:
Think the return value can be void instead, since the only implementation always returns true, and the return value is not checked either. Or is it for future extensions?

Reply:
this, and probably all of the currently void methods should ideally be made bool and return values checked. i'll add checking instead.

Reply:
done.